lib/fluent/logger/fluent_logger.rb

```diff
@@ -55,6 +57,9 @@ def initialize(tag_prefix = nil, *args)
         @host = options[:host]
         @port = options[:port]
 
+        @require_ack_response = options[:require_ack_response]
```

Comment:
Could you separate PR?
`require_ack_response` is not a main of `PackedForward` message.